src/state/sagas/scenario/SaveScenario/SaveScenario.js

```diff
@@ -35,6 +35,7 @@ export function* saveScenario(action) {
       scenario: { parametersValues: updateData.parametersValues, lastUpdate: updateData.lastUpdate },
     });
   } catch (error) {
+    console.error(error);
     yield put(
```

Comment:
Maybe it's possible to prevent to dispatch two error by customize the error message in the action ?